tests/unit_tests/pyanaconda_tests/modules/timezone/test_module_timezone.py

```diff
@@ -378,3 +382,62 @@ def test_geoloc_result_callback(self):
         result = GeolocationData.from_values(territory="", timezone="")
         self.timezone_module._set_geolocation_result(result)
         assert self.timezone_module.geolocation_result == result
+
+    @patch("pyanaconda.modules.timezone.timezone.get_all_regions_and_timezones")
+    def test_get_timezones(self, get_all_tz):
+        """Test getting a listing of all valid timezones."""
+        get_all_tz.return_value = OrderedDict([("foo", {"bar", "baz"})])
+        # as the timezones for a region are listed as a set, we need to be a bit careful
+        # when comparing the results
+        result = self.timezone_module.get_timezones()
+        assert list(result.keys()) == ["foo"]
+        assert sorted(result["foo"]) == ['bar', 'baz']
+        get_all_tz.assert_called_once()
```

Comment:
The test is good, but it would be best to replace
```
result = self.timezone_module.get_timezones()
```
with 
```
result = self.timezone_interface.GetTimezones()
```
and adjust the checking accordingly. Then, the other test for the interface would become redundant. Same for all the other new functions.

Reply:
Changed. :)